File: bank_console_python/dtos/acc_info.py
from dataclasses import dataclass
from typing import Optional


@dataclass
class AddresInfo:
    pin: Optional[int] = None
    city: Optional[str] = None
    state: Optional[str] = None


@dataclass
class AccInfo:
    name: Optional[str] = None
    date: Optional[str] = None
    balance: int = 0
    acc_no: int = 0
    addres: Optional[AddresInfo] = None

File: BankConsole1/Dtos/AccInfo.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace BankConsole1.Dtos
{
    public class AccInfo
    {
        public string? Name { get; set; }
        public string? Date { get; set; }
        public int Balance { get; set; }
        public int AccNo { get; set; }
        public AddresInfo? Addres { get; set; }
    }

    public class AddresInfo
    {
        public int? Pin { get; set; }
        public string? City { get; set; }
        public string? State { get; set; }
    }

    public class AddresInfotest
    {
        public int? Pin { get; set; }
        public string? City { get; set; }
        public string? State { get; set; }
    }
}
